website/docs/quick-start/installation/modal/index.md

```diff
@@ -103,6 +110,8 @@ def app():
             "8000",
             "--device",
             "cuda",
+            "--parallelism",
+            "4",
```

Comment:
please also set chat model (qwen-1.5B) for default modal deployment.

Reply:
add chat model: Qwen2-1.5B-Instruct